apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -165,15 +165,28 @@ defmodule Aecore.Chain.Worker do
     Enum.each(new_block.txs, fn(tx) -> Pool.remove_transaction(tx) end)
     new_block_hash = BlockValidation.block_header_hash(new_block.header)
 
-    updated_blocks_map = Map.put(blocks_map, new_block_hash, new_block)
+    updated_blocks_map  = Map.put(blocks_map, new_block_hash, new_block)
+    hundred_blocks_map  =
+    if Kernel.map_size(updated_blocks_map) > 100 do
```

Comment:
make the number configurable 

https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants